torch/csrc/profiler/kineto_client_interface.cpp

```diff
@@ -45,7 +47,10 @@ class LibKinetoClient : public libkineto::ClientInterface {
   }
 
   void stop() override {
-    (void)disableProfiler();
+    if (ProfilerStateBase::pop(/*global=*/true) ||
```

Comment:
`ProfilerStateBase::pop()` would not be enough here? Or, would like to show this if statement cover both cases?
```
  static std::shared_ptr<ProfilerStateBase> pop() {
    auto out = pop(/*global=*/true);
    return out ? out : pop(/*global=*/false);
  }
```